dev-python/numpy/python3_numpy-1.11.2.recipe

```diff
@@ -23,22 +23,33 @@ PROVIDES="
 REQUIRES="
 	haiku
 	cmd:python3
+	lib:libopenblas
 	"
 
 BUILD_REQUIRES="
 	haiku_devel
 	python3_setuptools
+	devel:libopenblas
 	"
 BUILD_PREREQUIRES="
 	cmd:python3
 	cmd:git
 	cmd:gcc
+	cmd:gfortran
 	"
 
 BUILD()
 {
+	rm -rf site.cfg || true
+	# Check at every update in site.cfg.example
+	echo "[openblas]" >> site.cfg
+	echo "libraries = openblas" >> site.cfg
+	echo "library_dirs = /system/`echo ${relativeDevelopLibDir}`" >> site.cfg
+	echo "include_dirs = /system/`echo ${relativeIncludeDir}`" >> site.cfg
+	echo "runtime_library_dirs = /system/`echo ${relativeLibDir}`" >> site.cfg
```

Comment:
Shouldn't this be done in the PATCH() function rather than BUILD() ? It would also make the hack with rm unneeded.